src/app.ts

```diff
@@ -6,16 +6,16 @@ import "@junto-foundation/junto-elements";
 import "@junto-foundation/junto-elements/dist/main.css";
 
 import { createPinia } from "pinia";
-
-import { ApolloClient, InMemoryCache } from "@apollo/client";
-import { WebSocketLink } from "@apollo/client/link/ws";
 import { Ad4mClient } from "@perspect3vism/ad4m";
+
 import { TimeoutCache } from "./utils/timeoutCache";
 import { ProfileExpression } from "./store/types";
+import { ApolloClient, InMemoryCache } from "@apollo/client";
+import { WebSocketLink } from "@apollo/client/link/ws";
 
 export const apolloClient = new ApolloClient({
   link: new WebSocketLink({
-    uri: `ws://localhost:4000/graphql`,
+    uri: `ws://localhost:5656/graphql`,
```

Comment:
is there a reason for changing the port?

Reply:
Yeah I pushed this by mistake will revert this back.